src/AudioFileSourceHTTPStream.h

```diff
@@ -52,14 +56,42 @@ class AudioFileSourceHTTPStream : public AudioFileSource
     enum { STATUS_HTTPFAIL=2, STATUS_DISCONNECTED, STATUS_RECONNECTING, STATUS_RECONNECTED, STATUS_NODATA };
 
   private:
-    virtual uint32_t readInternal(void *data, uint32_t len, bool nonBlock);
+    bool is_chunked;
+    std::size_t next_chunk;
```

Comment:
Not sure of the reason for `std::size_t` vs. the well-known stdint.h `size_t`?

Reply:
Does it really matter?